src/main/java/org/hydev/mcpm/client/installer/presenter/ResultPresenter.java

```diff
@@ -0,0 +1,14 @@
+package org.hydev.mcpm.client.installer.presenter;
+
+/**
+ * Interface for presenting the state of plugins
+ *
+ */
+public interface ResultPresenter {
```

Comment:
This doesn't present all results... right?

```suggestion
public interface InstallResultPresenter {
```

Reply:
I thought of having install, uninstall and update share the same interface with results since they all seem to have State. It also works if all of them pass the presenter to the use case, but I will leave it InstallResultPresenter for now XD